File: mvn-advent-of-code/src/test/java/advent/twenty_eighteen/Day25FourDimensionalAdventureTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day25FourDimensionalAdventureTest {
    @Test
    public void checkReal() {
        Day25FourDimensionalAdventure cc = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-input.txt");
        assertEquals(314, cc.getPart1Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day25FourDimensionalAdventure cc = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-example#1.txt");
        assertEquals(2, cc.getPart1Answer().intValue());
    }

    @Test
    public void checkExample2() {
        Day25FourDimensionalAdventure cc = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-example#2.txt");
        assertEquals(4, cc.getPart1Answer().intValue());
    }

    @Test
    public void checkExample3() {
        Day25FourDimensionalAdventure cc = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-example#3.txt");
        assertEquals(3, cc.getPart1Answer().intValue());
    }

    @Test
    public void checkExample4() {
        Day25FourDimensionalAdventure cc = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-example#4.txt");
        assertEquals(8, cc.getPart1Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day25FourDimensionalAdventureTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day25FourDimensionalAdventureTest  extends FlatSpec with Matchers {
  "2018 Day 25 - Input" should "calculate answers" in {
    val dp = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day25FourDimensionalAdventure("/twenty_eighteen/Day25-FourDimensionalAdventure-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
